baccmod/base_acceptance_map_creator.py

```diff
@@ -864,6 +868,41 @@ def _get_interpolated_background(self, interp_func: interp1d, zenith: u.Quantity
 
         return interp_bkg
 
+    def _interpolate_bkg_to_energy_axis(self, data_bkg: u.Quantity, energy_axis_computation: MapAxis):
+        """
+            Compute the final background model from the provided data by interpolating the energy axis used for computation to the ones for the model
+
+            Parameters
+            ----------
+            data_bkg : u.Quantity
+                The data cube of the background model. The energy axis need to be the first one.
+            energy_axis_computation : gammapy.maps.geom.MapAxis
+                The energy axis used for computation
+
+            Returns
+            -------
+            final_data_bkg : u.Quantity
+                The background data cube interpolated to the energy axis of the output model
+        """
+
+        unit = data_bkg.unit
+        raw_log_data = np.log10(data_bkg.value+100.*np.finfo(np.float64).tiny)
```

Comment:
I don't like biasing all your data as a default. Even if by a small quantity.

Reply:
the bias should be for any background value we expect smaller that the numerical precision of the floating point and therefore have no impact at all.
It will only bias value smaller than 10^-290.

Reply:
Since you use it to interpolate in log space, it biasses the results of your interpolation between zeros and values by possibly hundreds of order of magnitude.
And if there are no zeros then it has no effect.

So I guess it prevents crashes when zeros happen which is good (is it?), but I remain sceptical.